CHANGELOG.md

```diff
@@ -371,7 +371,7 @@ Major changes include:
 * Snowflake: Complete `LIKE` grammar [#2779](https://github.com/sqlfluff/sqlfluff/pull/2779) [@jpy-git](https://github.com/jpy-git)
 * Spark3: Auxiliary`FILE` and `JAR` statements [#2778](https://github.com/sqlfluff/sqlfluff/pull/2778) [@R7L208](https://github.com/R7L208)
 * Snowflake: Refine `SET`/`UNSET` `MASKING POLICY` grammar [#2775](https://github.com/sqlfluff/sqlfluff/pull/2775) [@jpy-git](https://github.com/jpy-git)
-* L049 bug: correct over zealous `=` --> `IS` [#2760](https://github.com/sqlfluff/sqlfluff/pull/2760) [@OTooleMichael](https://github.com/OTooleMichael)
+* L049 bug: correct over zealous `=` -> `IS` [#2760](https://github.com/sqlfluff/sqlfluff/pull/2760) [@OTooleMichael](https://github.com/OTooleMichael)
```

Comment:
Let's make a more unique identifier to avoid the need to do this by adding something like this to the appropriate place in the CHANGELOG.md file:

```html
<!--Start Of Releases (DO NOT DELETE THIS LINE)-->
```

Reply:
Nice idea. https://github.com/greg-finley/sqlfluff/pull/8